src/SFA.DAS.PR.Jobs/Functions/Notifications/SendNotificationsFunction.cs

```diff
@@ -64,13 +70,29 @@ public async Task Run([TimerTrigger("%SendNotificationsFunctionSchedule%", RunOn
         _logger.LogInformation("{FunctionName} - Processed {ProcessedCount} notifications.", nameof(SendNotificationsFunction), processedCount);
     }
 
-    private async Task<int> TryProcessProviderNotification(Notification notification, CancellationToken cancellationToken)
+    private async Task<int> TryProcessProviderNotification(Notification notification, FunctionContext executionContext, CancellationToken cancellationToken)
     {
         try
         {
-            ProviderEmailRequest providerEmailRequest = await CreateProviderEmailRequest(notification, cancellationToken);
-            await _pasAccountApiClient.SendEmailToAllProviderRecipients(notification.Ukprn!.Value, providerEmailRequest, cancellationToken);
-            notification.SentTime = DateTime.UtcNow;
+            switch(notification.NotificationType)
+            {
+                case nameof(NotificationType.Provider):
+                    {
+                        ProviderEmailRequest providerEmailRequest = await CreateProviderEmailRequest(notification, cancellationToken);
```

Comment:
This request object and the corresponding employer request object (ProviderEmailRequest and SendEmailCommand) are differently formatted and it feels a bit odd

Reply:
@PeterDavidWilliams Yea one goes off to the notifications message queue and one goes off to an external API, that's just the format of the required objects so no issue there.

Reply:
Ok, happy then, resolved